wp-cli/wp-cli
# Remove the pre-commit hook installation and its corresponding command

Fixes #5480

See https://github.com/wp-cli/wp-cli/issues/5859

This removes the script which sets up the pre-commit hook and its corresponding script.

If you have this pre-commit script installed, you can remove it by deleting the `.git/hooks/pre-commit` file.

## Steps for testing

1. Clone this repo
2. Switch to this branch
3. Run `composer install`
4. Confirm no pre-commit hook has been installed into `.git/hooks/pre-commit`
5. Commit some code that would normally fail coding standards checks
6. Confirm there's no error message and the commit succeeds
